aries_vcx/src/handlers/connection/inviter/states/invited.rs

```diff
@@ -10,16 +10,17 @@ pub struct InvitedState {
     pub invitation: Invitation,
 }
 
-impl From<(InvitedState, ProblemReport)> for NullState {
-    fn from((_state, _error): (InvitedState, ProblemReport)) -> NullState {
-        trace!("ConnectionInviter: transit state from InvitedState to NullState");
+// TODO: These have no justification for being here anymore
```

Comment:
Why is that? I see we still use this, for example  here
```
aries_vcx/src/handlers/connection/inviter/state_machine.rs:304:48
```

Reply:
The comment does not say it is not used, but that it has no justification for being in the folder dedicated to inviter state.